File: AOJ/itp_1_7_c.php
<?php

list($r, $c) = explode(" ", trim(fgets(STDIN)), 2);

for ($i = 0; $i < $r; $i++) {
    $row = explode(" ", trim(fgets(STDIN)));

    $sum = 0;
    foreach ($row as $j => $n) {
        $sum += $n;
        $arr[$j] += $n;
    }
    $row = $sum;
    printf("%s\n", implode(' ', $row));
}

$sum = array_sum($arr);
$arr[] = $sum;
printf("s\n", implode(' ', $arr));

File: AOJ/itp_1_7_c.py
r, c = map(int, input().split())
arr = []

for i in range(r):
    arr.append(list(map(int, input.split())))

for i in arr:
    line.append(sum(line))

arr.append([sum([line[i] for line in arr]) for i in range(c + 1)])

for line in arr:
    print(*line)
